st3/sublime_lib/view_stream.py

```diff
@@ -98,7 +103,7 @@ def readline(self, size=-1):
 
         return self._read(begin, end, size)
 
-    def _read(self, begin, end, size):
+    def _read(self, begin: int, end: int, size: int) -> str:
```

Comment:
```suggestion
    def _read(self, begin: int, end: int, size: Optional[int]) -> str:
```

Reply:
This can just be a mandatory `int` with no default because we're always passing a `int`.

Reply:
In that case we can remove the `None` check below.